frontend/src/pages/org/settings/components/profile.ts

```diff
@@ -0,0 +1,158 @@
+import { localized, msg } from "@lit/localize";
+import { serialize } from "@shoelace-style/shoelace/dist/utilities/form.js";
+import { html } from "lit";
+import { customElement } from "lit/decorators.js";
+
+import { BtrixElement } from "@/classes/BtrixElement";
+import { columns, type Cols } from "@/layouts/columns";
+import { OrgTab } from "@/routes";
+import { formValidator, maxLengthValidator } from "@/utils/form";
+
+@localized()
+@customElement("btrix-org-settings-profile")
+export class OrgSettingsProfile extends BtrixElement {
+  private readonly validateDescriptionMax = maxLengthValidator(150);
+
+  render() {
+    const orgHomeUrl = `${window.location.protocol}//${window.location.hostname}${window.location.port ? `:${window.location.port}` : ""}/orgs/${this.orgSlug}`;
+
+    const cols: Cols = [
+      [
+        html`
+          <label for="orgVisibility" class="form-label text-xs">
+            ${msg("Visibility")}
+          </label>
+          <div>
+            <sl-switch
+              id="orgVisibility"
+              name="enablePublicProfile"
+              size="small"
+              ?checked=${this.org?.enablePublicProfile}
+            >
+              ${msg("Allow anyone to see org")}
+            </sl-switch>
+          </div>
+        `,
+        msg(
+          "If enabled, anyone with the link to your org's Browsertrix URL will be able to view the profile page and public collections.",
+        ),
+      ],
+      [
+        html`
+          <sl-textarea
+            class="with-max-help-text"
+            name="publicDescription"
+            size="small"
+            label=${msg("Description")}
+            autocomplete="off"
+            value=${this.org?.publicDescription || ""}
+            minlength="2"
+            rows="2"
+            help-text=${this.validateDescriptionMax.helpText}
+            @sl-input=${this.validateDescriptionMax.validate}
+          ></sl-textarea>
+        `,
+        msg(
+          "Write a short description that introduces your organization or your web archive.",
+        ),
+      ],
+      [
+        html`
+          <sl-input
+            class="mb-2"
+            name="publicUrl"
+            size="small"
+            label=${msg("Website")}
+            value=${this.org?.publicUrl || ""}
+            minlength="2"
+            placeholder="https://"
```

Comment:
Can we auto-prepend `https://` if the user leaves it out?  This is a frequent cause for confusion on other fields, users aren't used to websites being as strict as we are and not fixing it automatically.

```suggestion
            placeholder="https://example.com"
```

Additionally / or, we need some error text telling people that the required format is `https://example.com` below the field so when it is in its error state they have more information on a resolution.

Reply:
Ilya just added auto-prepending https in the workflow editor, could crib the solution from that